### PR TITLE
Add --build-delimiter to the cli options

### LINKED ISSUE #1987: --build-delimiter to add to the cli options
We recently added `--build-delimiter`

https://github.com/webpack/webpack-cli/blob/master/bin/webpack.js#L207-L210

I can make a PR to add it to the documentation but I would like to know which section we should add it to.

### PR DESCRIPTION
**Pull request by ematipico**
It closes #1987 

It adds information about `--build-delimiter` options that can be passed to the `cli`

### CODE REVIEW

File: src/content/api/cli.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ Parameter                 | Explanation                                 | Input
 `--output-pathinfo`       | Include a comment with the request for every dependency | boolean | false
 `--output-public-path`    | The public path for the assets              | string     | /
 `--output-source-map-filename` | The output filename for the SourceMap  | string     | [name].map or [outputFilename].map
+`--build-delimiter` | Display custom text after build output | string | /
```
[the remark this answers is not in the document]
Comment (ematipico) [reply src/content/api/cli.md]: Yeah sure

**Comment by shellscape** [comment]
_I promise I'm not trolling this issue_

These kinds of CLI options add complexity and grow the schema without need. A reporter paradigm takes care of this (and an infinite number of terminal output options, capabilities). We need to have a serious conversation about the benefits of webpack-command, as it already provides that paradigm. Pointing users to that project versus adding more complexity to webpack-cli seems like it would be preferable.

**Comment by shellscape** [comment]
This will introduce complexity to webpack-cli that is not needed, and it will make the transition to another module more difficult for users in the future. One of the reasons I created webpack-command is because of these kinds of architectural decisions and not recognizing poor choices and additions that didn't need to be made. This will willingly add a feature that will instantly be obsolete under webpack-command. It boggles the mind.

**Comment by montogeek** [comment]
Thanks!

**Pull request merged**